src/BBT.StructureTools/Strategy/ICreateByBaseAsCriterionStrategy.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright © BBT Software AG. All rights reserved.
+
+namespace BBT.StructureTools.Strategy
+{
+    using BBT.StrategyPattern;
+
+    /// <summary>
+    /// Definition of a strategy where a specific implementation instance is created based on a
+    /// specific implementation it is depending on.
```

Comment:
This sentence is confusing and doesn't make sense in the last par (`... implementation it is depending on...`)